src/TestLogger.php

```diff
@@ -65,6 +65,27 @@ final class TestLogger extends AbstractLogger
     /** @var array<int|string, array<int, array<string, mixed>>> */
     public array $recordsByLevel = [];
 
+    /** @var array<string, string|int> */
+    public array $levelMap = [
```

Comment:
I think I'd prefer to make this `private` for now so that outside code can't mutate it after instantiation.

```suggestion
    private array $levelMap = [
```